client/my-sites/media-library/style.scss

```diff
@@ -44,6 +44,46 @@
 .media-library__header .button.media-library__upload-more {
 	margin-left: 0;
 }
+
+.media-library__filter-bar {
+	display: flex;
+	flex-flow: row nowrap;
+	align-items: stretch;
+	position: relative;
+
+	.section-nav {
+		flex: 1 auto;
+	}
+
+	.plan-storage {
+		display: none;
+		min-width: 200px;
+		margin-left: 1px;
+		margin-bottom: 9px;
+		box-shadow:
+		  0 0 0 1px transparentize( lighten( $gray, 20% ), .5 ),
+		  0 1px 2px lighten( $gray, 30% );
+		background-color: $white;
+		padding-left:20px;
+		padding-right: 20px;
+		box-sizing: border-box;
+		flex: 1 auto;
+
+		@include breakpoint( ">480px" ) {
+			display: flex;
+		}
+
+		@include breakpoint( ">660px" ) {
+			display: none;
```

Comment:
I'm a bit confused by this rule @retrofox, could you add a screenshots of the different breakpoints?